### DISCUSSION TITLE
Is possible to resize a GDI+ bitmap on memory?

### DISCUSSION THREAD

**Discussion by CA10C3SAR** [ImageMagick/ImageMagick#4155] [Help]
Is possible to resize a GDI+ bitmap currently in memory using ImageMagick?
There's no need to save the image to disk, just would like to resize it.

**Comment by dlemstra** [comment]
On what platform and in which programming language do you want to accomplish this?

**Reply by CA10C3SAR** [reply]
Win10, I'm currently using AutoHotkey, but c++ also would help.
What I'm trying is to reduce the size of the images, as they're kept/used while in memory. 
After some time there are too many images and it's using too much RAM.
I wonder if ImageMagick could resize as good as this website: https://tinypng.com/

**Reply by dlemstra** [reply]
This doesn't sound like an ImageMagick question. But more a question about how an image could be stored differently on the clipboard of Windows. You will probably need to ask that somewhere else.
